posts/tests.py

```diff
@@ -281,3 +295,66 @@ def test_delete_polls_author(self, persist_poll):
     def test_poll_invalid_args(self, user, description, expected_error):
         with pytest.raises(expected_error):
             Poll(author=user, description=description)
+
+
+# -------------------------------------------- PollFile tests --------------------------------------------
+
+@pytest.mark.django_db()
+class TestPollFile:
+    # Check that the PollFile values are the same as the PollFile inputs.
+    def test_new_poll_file_input_same_as_output(self, new_poll_file):
+        assert new_poll_file.poll.author.username == USERNAME
+        assert new_poll_file.poll.description == DESCRIPTION
+        assert new_poll_file.file == FILE1
+
+    # Check if the PollFile is saved in the database and accessible via its Poll.
+    @pytest.mark.django_db()
```

Comment:
I don't need there is a need to add ` @pytest.mark.django_db()` annotation  on the methods, only on the class

Reply:
> I don't need there is a need to add ` @pytest.mark.django_db()` annotation on the methods, only on the class

I Agree